core/test/unit/importer_spec.js

```diff
@@ -59,13 +77,11 @@ describe('Importer', function () {
             it('knows when to process a file', function (done) {
                 var testFile = {name: 'myFile.json', path: '/my/path/myFile.json'},
                     zipSpy = sandbox.stub(ImportManager, 'processZip').returns(Promise.resolve()),
-                    fileSpy = sandbox.stub(ImportManager, 'processFile').returns(Promise.resolve()),
-                    cleanSpy = sandbox.stub(ImportManager, 'cleanUp').returns(Promise.resolve());
+                    fileSpy = sandbox.stub(ImportManager, 'processFile').returns(Promise.resolve());
 
                 ImportManager.loadFile(testFile).then(function () {
                     zipSpy.calledOnce.should.be.false;
                     fileSpy.calledOnce.should.be.true;
-                    cleanSpy.calledOnce.should.be.true;
                     done();
                 });
```

Comment:
There are a bunch of these test blocks that are missing `.catch(done)` handlers.